src/table/table.scss

```diff
@@ -122,265 +249,341 @@
-      cursor: not-allowed;
 
-      &:hover {
+      &:hover > .iui-slot > .iui-more-options {
+        visibility: hidden;
+      }
+
+      img,
+      svg:not(.iui-radial),
+      .iui-user-icon {
+        filter: $iui-icons-color-multicolor-disabled;
+      }
+    }
+
+    &.iui-positive {
+      &,
+      + .iui-expanded-content {
         @include themed {
-          background-color: t(iui-color-background-1);
+          box-shadow: inset $iui-xxs 0 0 0 t(iui-icons-color-positive);
         }
 
-        > .iui-tables-slot > .iui-tables-menu {
-          visibility: hidden;
+        *::selection {
+          @include themed {
+            background-color: rgba(
+              t(iui-color-foreground-positive-rgb),
+              t(iui-opacity-4)
+            );
+          }
         }
       }
 
-      > .iui-tables-cell {
-        > img,
-        > svg,
-        > .iui-user-icon,
-        > .iui-user-icon > img {
-          filter: $iui-icons-color-multicolor-disabled;
+      .iui-status {
+        @include themed {
+          fill: t(iui-icons-color-positive);
         }
       }
     }
-  }
+    
+    &.iui-warning {
+      &,
+      + .iui-expanded-content {
+        @include themed {
+          box-shadow: inset $iui-xxs 0 0 0 t(iui-icons-color-warning);
+        }
 
-  .iui-tables-message-container {
-    @include themed {
-      color: t(iui-text-color-muted);
-      background-color: t(iui-color-background-2);
-    }
-    display: flex;
-    justify-content: center;
-    height: 100%;
-    padding: $iui-xl;
-  }
+        *::selection {
+          @include themed {
+            background-color: rgba(
+              t(iui-color-foreground-warning-rgb),
+              t(iui-opacity-4)
+            );
+          }
+        }
+      }
 
-  .iui-tables-paginator {
-    @include themed {
-      background-color: t(iui-color-background-3);
+      .iui-status {
+        @include themed {
+          fill: t(iui-icons-color-warning);
+        }
+      }
     }
-    user-select: none;
 
-    &:hover {
-      @include themed {
-        background-color: t(iui-color-background-3) !important;
+    &.iui-negative {
+      &,
+      + .iui-expanded-content {
+        @include themed {
+          box-shadow: inset $iui-xxs 0 0 0 t(iui-icons-color-negative);
+        }
+
+        *::selection {
+          @include themed {
+            background-color: rgba(
+              t(iui-color-foreground-negative-rgb),
+              t(iui-opacity-4)
+            );
+          }
+        }
       }
-    }
 
-    > .iui-content {
-      @include themed {
-        border-bottom: solid 1px t(iui-color-background-4);
+      .iui-status {
+        @include themed {
+          fill: t(iui-icons-color-negative);
+        }
       }
-      width: 100%;
-      display: flex;
-      justify-content: space-around;
-      align-items: center;
-      padding: $iui-table-density;
-      flex: 1;
+    }
 
-      > .iui-full-line {
-        flex: 1;
+    &.iui-expanded {
```

Comment:
At some point this class will start giving us problems

Reply:
Replacing with `.iui-row-expanded` in next push.